odk1-src/briefcase-using.rst

```diff
@@ -204,9 +217,25 @@ To get help about the command line operation:
 Pulling form data from Aggregate
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+- CLI flag: `-plla` or `--pull_aggregate`
+- Usage:
+
+  .. code-block:: console
+
+      $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
 
-  $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
+- Help section:
```

Comment:
As noted above...



I think there is a better (native to Sphinx or in a plugin) to document command line options,
similar to the way we document functions.

I will try to look into this.